backend/models/event.js

```diff
@@ -13,8 +13,8 @@ const eventSchema = new mongoose.Schema({
         type: Array,
         ref: 'Comment'
     },
-    date: {
-        type: Date,
+    StartDate: {
+        type: String,

```

Comment:
I think we should store `Date` instead of `String`. It would be much easier to filter these events when we have an `Date` object. Otherwise, we won't be able to run queries on MongoDB that involves dates